BPOR/BPOR.Domain/Migrations/20241126154919_Offline-journey-table-rename-setup.cs

```diff
@@ -0,0 +1,245 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+#pragma warning disable CA1814 // Prefer jagged arrays over multidimensional
+
+namespace Dynamo.Stream.Handler.Migrations
+{
+    /// <inheritdoc />
+    public partial class Offlinejourneytablerenamesetup : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_EmailCampaignParticipants_EmailCampaigns_EmailCampaignId",
+                table: "EmailCampaignParticipants");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_EmailCampaignParticipants_SysRefEmailDeliveryStatus_Delivery~",
+                table: "EmailCampaignParticipants");
+
+            migrationBuilder.DropTable(
```

Comment:
drop table?

Reply:
this is a ref data table, all data that will be in this table is in the [config file](https://github.com/PA-NIHR-CRN/dte-study-api/blob/rms/feature/CRNCC-2416-notify-letter-pre-campaigns/BPOR/BPOR.Domain/Entities/Configuration/CampaignDeliveryStatusConfiguration.cs)
this was left as is as there is no risk of data loss.

Reply:
thanks @Tkirkwood I was just writing this as a note to double check. Sounds good